### ISSUE TITLE
its seem, keras.backend.gather(boxes, top_indices) is missing

### ISSUE THREAD

**Issue by LastSamurai333** [fizyr/keras-retinanet#879]
https://github.com/fizyr/keras-retinanet/blob/367d0095da755405451a073eb7df4f59f2f0642e/keras_retinanet/layers/filter_detections.py#L93
[Code](https://github.com/fizyr/keras-retinanet/blob/367d0095da755405451a073eb7df4f59f2f0642e/keras_retinanet/layers/filter_detections.py#L93)

I am expecting 

```
boxes               = keras.backend.gather(boxes, indices)
boxes               = keras.backend.gather(boxes, top_indices)
```

Instead of 

`boxes               = keras.backend.gather(boxes, indices)
`

similar to labels,indices

Why is following line missing (According to me)
`boxes               = keras.backend.gather(boxes, top_indices)
`
